room.js

```diff
@@ -1,703 +0,0 @@
-                            play()
-                            break    
-                        
-                        case 'open box':
-                        case 'open chest':
-                            await room1_boarded_box.open()
-                            play()
-                            break 
-
-                        case 'secret room':                         // Test visting room 2
-                            player.current_room = 'room2'
-                            play()
-                            break    
-                        
-                        case 'move':
-                            move(player.current_facing)
-                            break
-
-                        default:
-                            console.log("\nUnexpected answer!")
-                            play()
-                            break
-                        }
-                    break
-                
-                case 'east':
-                    console.log(room1.east_description)      // Room with the drawer
-                    answer = await ask("Enter action ")
-                    switch(answer){
-                        case 'i': 
-                        case 'check inventory': 
-                        case 'inventory':
-                        case 'inv': 
-                        case 'check inv':
-                            player.check_inventory()
-                            play()
-                            break
-
-                        case 'gargle':
-                            console.log("Sorry, I don't know how to gargle!")
-                            break   
-
-                        case 'open drawer':
-                        case 'grab drawer':
-                            room1_drawer.open()
-                            play()
-                            break
-
-                        case 'close drawer':
-                        case 'shut drawer':
-                            room1_drawer.close()
-                            break
-
-                        case 'examine drawer':
-                        case 'look at drawer':
-                        case 'observe drawer':
-                            examine(room1_drawer)
-                            play()
-                            break
-                        
-                        case ('read note' || 'examine note'):                         // Test visting room 2
-                            room1_drawer.read()
-                            play()
-                            break    
-                        
-                        case 'move':
-                            move(player.current_facing)
-                            break
-
-                        default:
-                            console.log("Unexpected answer!")
-                            play()
-                            break
-                        }
-                        
-                    break
```

Comment:
By breaking away from 'switch' statements, I believe this code can be condensed quite a bit through using 'if' or 'else if' statements. as well as breaking your inventory out of each individual switch statement.